src/functionalTest/java/uk/gov/hmcts/reform/bulkscanprocessor/controllers/ProcessedEnvelopeMessageHandlingTest.java

```diff
@@ -114,7 +114,7 @@ private EnvelopeResponse getEnvelope(String zipFilename) {
 
     private void sendProcessedEnvelopeMessage(UUID envelopeId) throws Exception {
         IMessage message = new Message(
-            objectMapper.writeValueAsString(new ProcessedEnvelope(envelopeId))
+            objectMapper.writeValueAsString(new ProcessedEnvelope(envelopeId, "2133333", "EXCEPTION_RECORD"))
```

Comment:
for the sake of argument since we don't validate those, can we stick in `ccd-reference` and `ccd-type` instead? or have comments next to them in a multiline object creation structure ;)

Reply:
I think it is better to remove objectMapper as well.

Reply:
and are we ok with string  casetype instead of enum